common/globalTruffleConfig.js

```diff
@@ -126,6 +126,8 @@ addLocalNetwork(networks, "coverage", { port: 8545, network_id: 1234 });
 // MetaMask truffle provider requires a longer timeout so that user has time to point web browser with metamask to localhost:3333
 addLocalNetwork(networks, "metamask", { provider: new MetaMaskTruffleProvider(), networkCheckTimeout: 500000 });
 
+addLocalNetwork(networks, "mainnet-fork", { port: 8545, network_id: 1 });
```

Comment:
This change was added to strictly enforce the `network_id` of 1 when running a ganache fork of the main net. This ensures that when running any kind of simulation or main net fork you explicitly use the main net contracts, forked locally.

Reply:
SGTM